src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+  
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+  
+.form {
+  margin-top: 20px;
+}
+  
+.first_search {
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+  
+.first::before {
+  content: "";
+  display: block;
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  margin-top: 25px;
+  margin-left: 26px;
+  background-image: url(./images/Search.svg);
+}
+  
+.second_search {
+  height: 42px;
+  width: 100%;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+  
+.second::before {
+  content: "";
+  display: block;
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  margin-top: 15px;
+  margin-left: 13px;
+  background-image: url(./images/Search.svg);
+  background-size: cover;
+}
```

Comment:
Don't repeat yourself. Properties position, display, background-image, and width are common for both bars, imgs. Move it to common class